common/public/include/SessionServerCommon.h

```diff
@@ -73,9 +74,12 @@ struct AppConfig
  */
 struct ServerManagerConfig
 {
-    std::list<std::string> sessionServerEnvVars{}; /* List of environment variables, that need to be passed to
-                                                      RialtoSessionServer */
-    int numOfPreloadedServers{0};                  /* Number of preloaded servers */
+    std::list<std::string> sessionServerEnvVars{};          /* List of environment variables, that need to be passed to
+                                                               RialtoSessionServer */
+    int numOfPreloadedServers{0};                           /* Number of preloaded servers */
```

Comment:
nit pick: I think that uint would be better type for numOfPreloadedServers

Reply:
done